src/DotNetStatus/JsonData/AnalyzerInfo.cs

```diff
@@ -0,0 +1,37 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.Text.RegularExpressions;
+
+namespace Microsoft.Fx.Portability.ObjectModel
+{
+    public class DiagnosticAnalyzerInfo
+    {
+        public string AnalyzerName { get; set; }
+        public string Id { get; set; }
+        public ICollection<CompatibilityRangeAttribute> Attributes { get; set; }
+        public bool IsCompatibilityDiagnostic { get; set; }
+
+        public int IdNumber
+        {
+            get
+            {
+                var matches = Regex.Match(Id, @"(\d+)");
```

Comment:
@JJVertical: You are correct, there is a start-up cost to compile the Regex. How often will this Regex be used? I guess if it is many times over, it'd be good to compile it. Otherwise, not. 
http://stackoverflow.com/a/7707369